PDSim/scroll/core.py

```diff
@@ -2011,34 +2074,39 @@ def lump_energy_balance_callback(self):
             # and the motor efficiency as a function of the torque [N-m]
             eta, omega = self.motor.apply_map(self.tau_mechanical)
             self.eta_motor = eta
-            self.omega = omega
+            self.omega = omega*(1-self.slip_ratio)
```

Comment:
I think this is a non-backwards-compatible change, right?  If ``slip_ratio`` is not provided, then the code fails?

Reply:
I forgot to remove this. I was playing around with validating a motor map.